VAMobile/src/screens/BenefitsScreen/Letters/GenericLetter/GenericLetter.tsx

```diff
@@ -51,9 +59,9 @@ function GenericLetter({ navigation, route }: GenericLetterProps) {
     <Box mb={theme.dimensions.contentMarginBottom}>
       {letterType === LetterTypeConstants.serviceVerification && (
         <Box mb={theme.dimensions.standardMarginBetween}>
-          <AlertBox border="informational">
+          <AlertWithHaptics variant="info">
             <TextView variant="MobileBody">{t('letters.serviceVerificationLetter.informational')}</TextView>
```

Comment:
Could potentially be simplified moving this to the `description` prop to AlertWithHaptics.